__tests__/api.test.js

```diff
@@ -317,3 +317,35 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+describe("DELETE /api/comments/:comment_id", () => {
+  it("removes a comment by comment id", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then(() => {
+        return db.query(`
+        SELECT * FROM comments
+        WHERE comment_id = 1
+        `);
+      })
+      .then((result) => {
+        expect(result.rows.length).toBe(0);
+      });
+  });
+  it("returns 404 when the comment_id does not exist", () => {
+    return request(app)
+    .delete("/api/comment/99999")
```

Comment:
this test has passed because it's invalid endpoint (should be /api/comments/99999)